### ISSUE TITLE
Debug `console.log` shows up in the code published to NPM

### ISSUE THREAD

**Issue by akphi** [maslianok/react-resize-detector#231]
Hi,

I started noticing that debug `console.log` message being left in the code published to NPM as of version `8.0.2`, `8.0.1` doesn't have this problem.

https://www.npmjs.com/package/react-resize-detector?activeTab=explore
See `index.js` line 282-283

```js
entries.forEach(function (entry) {
                console.log(entry && entry.contentRect, entry.contentBoxSize);
```

I don't find this in the code though. Could you take a look @maslianok . Thanks!

**Comment by snelsi** [comment]
I assume @maslianok was just testing some stuff related to #220 locally on his computer and forgot to remove local changes before the release. 🤔

There are no `console.log` in the repo's codebase

**Comment by Robloche** [comment]
I noticed the same thing after having updated to 8.0.2.

**Comment by maslianok** [comment]
Sorry for this guys :) Fixed in v8.0.3

**Comment by maslianok** [comment]
> I assume @maslianok was just testing some stuff related to #220 locally on his computer and forgot to remove local changes before the release. 🤔
> 
> There are no `console.log` in the repo's codebase

correct 🤣
